README.md

```diff
@@ -199,8 +221,23 @@ monitoring dashboards, etc. see the [Deployments README](./deployments/README.md
 
 ### How to send a message
 
-A straightforward way to interact with and test the bridge is sending messages. This is explained
-in the [send message](./docs/send-message.md) document.
```

Comment:
This document still contains useful info, please leave a reference to it.

You can expand on this by saying something like: In this section we'll show you how to quickly send a bridge message, if you want to see more details read the "send message" docs.